src/demo/index.ejs

```diff
@@ -7,8 +7,17 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
-
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
```

Comment:
Why did you cancel the link to the fonts? Was that intended?

Reply:
yes, otherwise it has to be added to the CSP. Also it's not in the sdk, so it can mislead what the sdk will look like in the end.

Reply:
unless it is in the sdk, do you know?

Reply:
As discussed, this is not in the SDK which means that we are only seeing the right font in the demo app. We need to fix this as part of another story